<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_employer</class>
 <widget class="QWidget" name="gestion_employer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#FFFFFF</string>
   </property>
   <property name="title">
    <string>Gestion Employés</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1351</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter et consulter</string>
     </attribute>
     <widget class="QPushButton" name="Supprimer">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>449</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_ROLE_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>419</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agenst Des Attractions</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agents De Guichet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femmes De Ménage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agents Securité</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="TRI">
      <property name="geometry">
       <rect>
        <x>704</x>
        <y>549</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri par ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>293</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PRENOM">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>299</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;HEURE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>469</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EXCEL">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>539</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>EXCEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>419</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ROLE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;hhh&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>539</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_ROLE">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>419</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agenst Des Attractions</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agents De Guichet</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femmes De Ménage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agents Securité</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agents de Maintenance</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Supprimer">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>469</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>249</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NBRJ">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>359</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>349</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>nombre de jour
travaillé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modifier">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>469</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TRIP">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>549</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri par Prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>489</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche par ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TRIN">
      <property name="geometry">
       <rect>
        <x>824</x>
        <y>549</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri par Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ID">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>189</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NOM">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>249</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>189</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Camera</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>551</width>
        <height>251</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>549</width>
         <height>249</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="OptionsPushButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>options</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ard">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>373</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Access&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
